training/distributed_training/pytorch/model_parallel/gpt2/smp-fine-tune-gpt-sharded-data-parallel.ipynb

```diff
@@ -0,0 +1,971 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        "offload_activations": 0,
</pre><p>below this line, why you don't keep <code>activation_loading_horizon</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
If offload_activations is off then we don't care about activation_loading_horizon